Cancel imput password fixed

hyt47 opened the pull request:
Application.Exit(); gives an error so I replaced it with Environment.Exit(0);

Jessecar96 commented:
Mmmm it looks like this isn't the proper way to close the whole program. If Application.Exit() isn't working right then there's another problem somewhere else.
http://stackoverflow.com/a/13046082/1385710

hyt47 commented:
That error was my bad, I added a piece of code that was unnecessary and because of that I was getting an error. I removed that code. `Application.Exit();` works there.
